README.md

```diff
@@ -61,6 +63,45 @@ export default () => {
 };
 ```
 
+### with TypeScript
+
+```ts
+import { useFetch } from 'use-fetch-hooks';
+
+interface Data {
+  data: {
+    name: string;
+    color: string;
+  };
+}
+
+export default () => {
+  const { data, loading, error } = useFetch<Data>({
```

Comment:
Wanna keep this consistent through our codebase 😬 

alpha
```ts
const { data, error, loading } = useFetch<Data>({
    url: 'https://example/api',
  });
```

Reply:
If that's the case it should be a lint/prettier config.

Reply:
Hm, would have to look into to see if that's possible

Reply:
Updated those examples to use alphabetical destructuring.

Could use something like [this](https://www.npmjs.com/package/eslint-plugin-sort-destructure-keys) if you want to enforce this via linting.

IMO it's kinda counterintuitive - feel like destructured properties should be ordered by importance vs. alphabetically.